### PR TITLE
[11.0][MIG] stock_orderpoint_manual_procurement_uom

### PR DESCRIPTION
**Pull request by bodedra**
# Depends on

- [x] https://github.com/OCA/stock-logistics-warehouse/pull/385
- [x] https://github.com/OCA/stock-logistics-warehouse/pull/386
- [x]  https://github.com/OCA/stock-logistics-warehouse/pull/389
- [x]  https://github.com/OCA/stock-logistics-warehouse/pull/391
- [x]  https://github.com/OCA/stock-logistics-warehouse/pull/396

### CODE REVIEW

**Comment by rgarnau** [comment]
@bodedra - It's failing because you need to add the oca dependencies file referencing the PRs.

File: stock_orderpoint_manual_procurement_uom/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "category": "Warehouse Management",
+    "depends": [
+        "stock_orderpoint_uom",
+        "stock_orderpoint_manual_procurement",
+    ],
+    "data": [],
+    "license": "AGPL-3",
+    'installable': True,
+    'application': False,
+    'auto-install': True,
```
Comment (guewen) [inline stock_orderpoint_manual_procurement_uom/__manifest__.py]: `auto_install`

File: stock_orderpoint_manual_procurement_uom/wizards/make_procurement_orderpoint.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    @api.model
+    def _prepare_item(self, orderpoint):
+        vals = super(MakeProcurementOrderpoint, self)._prepare_item(orderpoint)
+        if orderpoint.procure_uom_id:
+            product_uom = orderpoint.procure_uom_id
+            vals['uom_id'] = product_uom.id
+        return vals
+
+    @api.multi
+    def make_procurement(self):
```
Comment (guewen) [inline stock_orderpoint_manual_procurement_uom/wizards/make_procurement_orderpoint.py]: This is a full override of the method from stock_orderpoint_manual_procurement without using super (https://github.com/OCA/stock-logistics-warehouse/pull/385/files#diff-caa1ed47e650d4d52dc2c98f85bee472R47).
Please extract the arguments passed to the `run` method in a `_prepare_procurement_group` method in the parent addon in https://github.com/camptocamp/qoqa_openerp/blob/master/odoo/local-src/speedy_views/models/procurement.py#L15-L22 so it allows you to extend this new method here and change  the quantity with _compute_quantity without replacing the parent method.

**Review by guewen — CHANGES_REQUESTED** [review]

File: stock_orderpoint_manual_procurement_uom/wizards/make_procurement_orderpoint.py (hunk 2 above)
Comment (bodedra) [reply stock_orderpoint_manual_procurement_uom/wizards/make_procurement_orderpoint.py]: @guewen Second link gives me "Page not found"

File: stock_orderpoint_manual_procurement_uom/__manifest__.py (hunk 1 above)
Comment (bodedra) [reply stock_orderpoint_manual_procurement_uom/__manifest__.py]: @guewen should I remove `auto_install` ?

File: stock_orderpoint_manual_procurement_uom/wizards/make_procurement_orderpoint.py (hunk 2 above)
Comment (bodedra) [reply stock_orderpoint_manual_procurement_uom/wizards/make_procurement_orderpoint.py]: @guewen I have removed `make_procurement()` method and set logic in `_prepare_procurement_values`.

**Comment by max3903** [comment]
@bodedra Can you rebase this PR? Thanks!

**Comment by max3903** [comment]
@bodedra I rebased your branch on OCA/11.0 to get the dependencies but a test is failing. Let me know if you need help to fix it.

**Comment by bodedra** [comment]
@max3903 following two PR will be resolve quantity issue.

https://github.com/OCA/stock-logistics-warehouse/pull/389
https://github.com/OCA/stock-logistics-warehouse/pull/391

**Comment by bodedra** [comment]
@guewen Would you please update your review with  my last comments and fixes. Thank you.

**Comment by bodedra** [comment]
Please review depends fixes at https://github.com/OCA/stock-logistics-warehouse/pull/396

**Comment by LoisRForgeFlow** [comment]
@bodedra could you please rebase this one now?

**Review by gurneyalex — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
@bodedra I just proposed you a PR to fix the tests: https://github.com/ursais/stock-logistics-warehouse/pull/1/

Could you please check?

**Comment by LoisRForgeFlow** [comment]
needs fixing label while the tests are red

**Comment by bodedra** [comment]
@lreficent Can you update your review ? It seems all good now. Appreciate your support. :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
Great, thanks @bodedra 

LTGM :+1:

**Comment by LoisRForgeFlow** [comment]
@guewen could you update your review?

**Pull request merged**